merge/map.go

```diff
@@ -0,0 +1,18 @@
+package merge
+
+import "maps"
+
+// MapStringString creates a new map and loads it from map args
+// this function takes a least 2 args and later map args take precedence.
+func MapStringString(m1 map[string]string, mapArgs ...map[string]string) map[string]string {
```

Comment:
it looks like by using a combination of `maps.Insert` and `maps.All` it is possible to do the merge too. Not sure if more optimal, just pointing it out: https://pkg.go.dev/maps#All

Reply:
Good point, I can see the argument for just using `maps`, but I think the one liner here is still moderately useful.